UnitTesting-master/pom.xml

```diff
@@ -31,6 +31,12 @@
             <artifactId>testng</artifactId>
```

Comment:
Lt's remove testng dependency. Adding dependencies that you do not use - not a good practice